### ISSUE TITLE
Global Attributes for classNames not working?

### ISSUE THREAD

**Issue by 5chub1** [caplin/FlexLayout#191]
Hey, @nealus I'm not quite sure if the global attributes for classnames like *'tabClassName'* and *'tabSetClassNameHeader'* are working correctly. Maybe I'm using them improperly? My understanding is, that those attributes need a string or array of strings, with classnames that should apply on all tabs or tabsets. As I'm working with TypeScript maybe typings for the Jsonmodel could help loads :) Thank you 👍

**Comment by nealus** [comment]
Yes, you are right, those attributes do not inherit from a global value like the others.

Will fix in next update.
